app/assets/javascripts/scoring.js

```diff
@@ -2,8 +2,9 @@ $(document).ready(function(){
   $('#score_form').areYouSure();
 
   $('.scoring').on('click', 'button.increment', function(){
-    $(this).siblings('button.increment').prop('disabled', false);
     var field_name = $(this).data('field');
+    var other_button = document.querySelectorAll("[data-field='" + field_name + "']");
+    other_button.forEach(function(element){ element.disabled = false});
```

Comment:
So, It's true that [you don't necessarily need to use jQuery](https://youmightnotneedjquery.com/); this is exactly how you would do this without it. And, for new projects, I am inclined _not_ to bring jQuery in because of the size and support burden.

However, in the context of a jQuery callback fuction, I think it's OK to do it the jQuery way:

```suggestion
    $("[data-field='" + field_name + "']").prop('disabled', false);
```